packages/hoppscotch-common/src/pages/index.vue

```diff
@@ -166,6 +175,20 @@ const removeTab = (tabID: string) => {
   }
 }
 
+const openReqRenameModal = () => {
+  showRenamingReqNameModal.value = true
+  reqName.value = currentActiveTab.value.document.request.name
```

Comment:
`reqName` seems to be set as an empty string in some instances.. can you look into this.
Here's a screen record:
[51072222-f3bd-4026-ad9b-de4faaf43776.webm](https://github.com/hoppscotch/hoppscotch/assets/10395817/662ec39f-bc47-4686-b132-02dc3da430a8)